tests/test_syslog_settings.rb

```diff
@@ -64,4 +67,38 @@ def test_create
                   )
     end
   end
+
+  def test_console
+    syslog_setting = Cisco::SyslogSettings.new('default')
+
+    assert_equal(2, syslog_setting.console)
+    syslog_setting.console = '5'
```

Comment:
better to use `syslog_setting.console = '1'`.  You will get the following error if you use a value of `5` and the baud rate is not compatible.

```shell
Node under test:
  - name  - dt-n9k5-1
  - type  - N9K-C9396PX
  - image - bootflash:///nxos.7.0.3.I6.1.bin

TestSyslogSettings#test_source_interface = 14.70 s = .
TestSyslogSettings#test_timestamp = 9.58 s = .
TestSyslogSettings#test_monitor = 10.89 s = .
TestSyslogSettings#test_console = 9.68 s = E

Finished in 44.858746s, 0.0892 runs/s, 0.3121 assertions/s.

  1) Error:
TestSyslogSettings#test_console:
Cisco::CliError: [#<Cisco::SyslogSettings:0x007f0d6b1a6038>] The command ' logging console 5' was rejected with error:
Baud rate of console should be 38400 to increase severity level

    /nobackup/mwiebe/NU/shermdog/cisco-network-node-utils/lib/cisco_node_utils/client/nxapi/client.rb:298:in `handle_output'
    /nobackup/mwiebe/NU/shermdog/cisco-network-node-utils/lib/cisco_node_utils/client/nxapi/client.rb:229:in `req'
    /nobackup/mwiebe/NU/shermdog/cisco-network-node-utils/lib/cisco_node_utils/client/nxapi/client.rb:130:in `set'
    /nobackup/mwiebe/NU/shermdog/cisco-network-node-utils/lib/cisco_node_utils/node.rb:276:in `set'
    /nobackup/mwiebe/NU/shermdog/cisco-network-node-utils/lib/cisco_node_utils/node.rb:216:in `config_set'
    /nobackup/mwiebe/NU/shermdog/cisco-network-node-utils/lib/cisco_node_utils/node_util.rb:72:in `config_set'
    /nobackup/mwiebe/NU/shermdog/cisco-network-node-utils/lib/cisco_node_utils/syslog_settings.rb:58:in `console='
    tests/test_syslog_settings.rb:75:in `test_console'

4 runs, 14 assertions, 0 failures, 1 errors, 0 skips
```